resources/design-document.md

```diff
@@ -99,8 +99,8 @@ String name;
 Double stars;
 String location;
 String Description;
-Rating rating;
-Style style;
+String rating;
```

Comment:
These should stay as Rating/Style (same below for Climb).

The model should be "pure Java" - i.e. if you have a type that's Rating, use that in the model. What you store in Dynamo will be a little different - typically a string representation of that type that's serialized/deserialized as you go back and forth (take a look at how songs are stored in the playlist in the sample project). It's a little odd to think through, the java model is "the same" as the dynamo data, but not identical because of this.